tests/test_filepath.py

```diff
@@ -18,34 +19,34 @@ def test_filepath(store="repo"):
 
     # create a mock file
     relpath = 'one/two/three'
-    os.makedirs(os.path.join(stage_path, relpath), exist_ok=True)
-    managed_file = os.path.join(stage_path, relpath, filename)
+    managed_file = Path(stage_path, relpath, filename)
+    managed_file.parent.mkdir(parents=True, exist_ok=True)
     data = os.urandom(3000)
-    with open(managed_file, 'wb') as f:
+    with managed_file.open('wb') as f:
         f.write(data)
 
     # put the same file twice to ensure storing once
-    uuid1 = ext.fput(managed_file)
-    uuid2 = ext.fput(managed_file)   # no duplication should arise if file is the same
+    uuid1 = ext.upload_filepath(managed_file)
+    uuid2 = ext.upload_filepath(managed_file)   # no duplication should arise if file is the same
     assert_equal(uuid1, uuid2)
 
     # remove to ensure downloading
-    os.remove(managed_file)
-    assert_false(os.path.isfile(managed_file))
+    managed_file.unlink()
+    assert_false(managed_file.exists())
 
     # Download the file and check its contents. Repeat causes no download from remote
     for _ in 1, 2:
-        restored_path, checksum = ext.fget(uuid1)
+        restored_path, checksum = ext.download_filepath(uuid1)
         assert_equal(restored_path, managed_file)
         assert_equal(checksum, dj.hash.uuid_from_file(managed_file))
 
     # verify same data
-    with open(managed_file, 'rb') as f:
+    with managed_file.open('rb') as f:
         synced_data = f.read()
     assert_equal(data, synced_data)
 
     # cleanup
-    ext.delete()
+    ext.delete(delete_external_files=True)
 
 
 def test_filepath_s3():
```

Comment:
This test failed on WIN10. For example, the result of `managed_file` was `dj/repo/one\two\three\picture.dat`.